newsroom/agenda/views.py

```diff
@@ -145,6 +151,11 @@ async def get_view_data() -> Dict:
     check_user_has_products(user, products)
     ui_config_service = UiConfigResourceService()
 
+    async def has_agenda_featured_items():
+        async for _ in FeaturedService().get_all():
+            return True
+        return False
```

Comment:
I think it might be more efficient to use a count_documents instead. Maybe like this below:

```python
has_featured_items = (await FeaturedService().mongo_async.count_documents({}) > 0)
```

@MarkLark86 what do you think?

Reply:
I've created a [PR](https://github.com/superdesk/superdesk-core/pull/2739) to add a count function on async resource services, that works for both MongoDB and Elasticsearch.
@eos87 Could you please review that, then @BrianMwangi21 can use it here instead

Reply:
Approved and merged already. @BrianMwangi21 you can pull the latest from superdesk-core and use the new method added by Mark. Thanks guys!